.circleci/config.yml

```diff
@@ -113,147 +125,106 @@ build_and_deploy: &build_and_deploy
           git rm --cache  web/sites/default/settings.local.php --ignore-unmatch
           git rm -r --cache web/sites/default/files --ignore-unmatch
 
-          # Set the git commit user name and email address so it's clear in Pantheon log who made the change
-          if [[ $CIRCLE_USERNAME = "" ]]; then
-            git config --global user.name Dependabot
-          else
-            git config --global user.name $CIRCLE_USERNAME
-          fi
-          git config --global user.email $eGov_Email
-
           export COMMIT_MSG="$(git log -1 --pretty=%B)"
           git commit -m "CircleCI deployment for: " -m "$COMMIT_MSG" --quiet
 
           # When commit on master, we deploy to DEV
           if [[ $CIRCLE_BRANCH = "master" ]]; then
-            echo "Deploy to Pantheon DEV site"
-            # Set Pantheon environment name
             echo "export PANTHEON_ENV=dev" >> $BASH_ENV && source $BASH_ENV
-            # Set the connection type to GIT
-            terminus connection:set $CIRCLE_PROJECT_REPONAME.dev git
-            git push -f pantheon master
-            # TODO: Sanitize DB here
-            # terminus env:clone-content $CIRCLE_PROJECT_REPONAME.test dev
+          else
+            echo "export PANTHEON_ENV=$CIRCLE_BRANCH" >> $BASH_ENV && source $BASH_ENV
+          fi
+
+          # Check if the multidev branch site exists. If not, create it
+          terminus multidev:list --format=list --field=Name $CIRCLE_PROJECT_REPONAME > ./env_list.txt
```

Comment:
This will try to create a multidev site named "dev" for commits to master now that this code block executes for all builds. I think this code block for checking for existing multidevs and creating them still needs to be within the "else" that only runs for commits that aren't on master.

Reply:
Great catch! Moved the site creation logic into "not master" block.